tests/dummy/app/router.js

```diff
@@ -1,7 +1,8 @@
-import Ember from 'ember'
 import config from './config/environment'
+import Ember from 'ember'
+const {Router: EmberRouter} = Ember
 
-var Router = Ember.Router.extend({
+var Router = EmberRouter.extend({
```

Comment:
Notice the variable here is named `Router`. Either the import or this variable which is exported needs renamed.